packages/deploy-slack-bot/src/index.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable max-len */
+import 'isomorphic-fetch'
+import crypto from 'crypto'
+import serverless from 'serverless-http'
+import express, { Request, Response, NextFunction } from 'express'
+import bodyParser from 'body-parser'
+import cors from 'cors'
+import { Context, APIGatewayProxyEvent } from 'aws-lambda'
+
+const app = express()
+const expressApp = serverless(app)
+
+export const queryParams = (params: Object) => {
+  return Object.keys(params)
+    .filter(key => params[key] !== undefined && params[key] !== null && params[key] !== '')
+    .map(key => {
+      if (Array.isArray(params[key])) {
+        return params[key].map((value: any) => `${key}=${value}`).join('&')
+      }
+      return `${key}=${params[key]}`
+    })
+    .join('&')
+}
+
+export const sendMessageToSlack = async (message: string) => {
+  const slackHook = process.env.SLACK_BOT_HOOK as string
+  const result = await fetch(slackHook, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      text: message,
+    }),
+  })
+  console.info('sendMessageToSlack', result.statusText)
+  return result.status
+}
+
+export const isAuthenticated = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
This file is a bit dense, could use a couple of comments!

Reply:
https://api.slack.com/authentication/verifying-requests-from-slack
It's very hard to say because it's like ad-hoc case for slack, so I'll put this link as comment on it